x/poe/contract/tgrade_ap_voting.go

```diff
@@ -11,6 +11,75 @@ type APVotingInitMsg struct {
 	GroupContractAddress string `json:"group_addr"`
 	// Dispute cost on this contract
 	DisputeCost sdk.Coin `json:"dispute_cost"`
-	// Waiting period for this contract
+	// Waiting period in seconds for this contract
 	WaitingPeriod uint64 `json:"waiting_period"`
+	// Cw3-fixed-multisig contract code id.
+	// This code id will be used to instantiate a group contract with the proposed arbiters list.
+	MultisigCodeID uint64 `json:"multisig_code_id"`
+}
+
+// APVotingExecute ap-voting contract execute messages
```

Comment:
Afaik, beside the contract setup all interactions with AP would be via frontend. 
It does not hurt to have some bindings here but do you have some concrete plans for them?